tests/acceptance/helpers/webdavHelper.js

```diff
@@ -167,3 +167,28 @@ exports.createFile = function (user, fileName, contents = '') {
     .then(res => httpHelper.checkStatus(res, `Could not create the file "${fileName}" for user "${user}".`))
     .then(res => res.text())
 }
+
+/**
+ * Get file or folder properties using webDAV api.
+ *
+ * @param {string} path
+ * @param {string} userId
+ * @param {array} properties
+ * @param {number} folderDepth
+*/
+exports.getProperties = function (path, userId, properties, folderDepth = 1) {
+  return new Promise((resolve) => {
+    exports.propfind(`/files/${userId}${path}`, userId, userSettings.getPasswordForUser(userId), properties,
+      folderDepth)
+      .then(str => {
+        const response = JSON.parse(convert.xml2json(str, { compact: true }))['d:multistatus']['d:response']
+        const items = []
+        response.map(item => {
+          items.push({
+            privatelink: item['d:propstat']['d:prop']['oc:privatelink']
```

Comment:
should be modified to get the properties not only the privatelinks, or function needs to be renamed

Reply:
done